examples/exampleapp/exampleapp_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) The EfficientGo Authors.
+// Licensed under the Apache License 2.0.
+
+package exampleapp
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/efficientgo/e2e"
+	e2edb "github.com/efficientgo/e2e/db"
+	e2einteractive "github.com/efficientgo/e2e/interactive"
+	"github.com/efficientgo/tools/core/pkg/backoff"
+	"github.com/efficientgo/tools/core/pkg/testutil"
+)
+
+func TestExampleApp(t *testing.T) {
+	// Start a new Docker environment.
+	e, err := e2e.NewDockerEnvironment("e2e_example_app")
+	testutil.Ok(t, err)
+	// Make sure resources are cleaned up.
+	t.Cleanup(e.Close)
```

Comment:
Shouldn't this be just below line 20? I otherwise in case there are errors, I believe `testutil.Ok(t, err)` is going to leave leftover, as under the hood it uses `testing.TB.FailNow`.

Reply:
Good catch :+1: